rst2pdf/tests/conftest.py

```diff
@@ -216,7 +216,21 @@ def runtest(self):
         # verify results
 
         if retcode:
-            self._fail('Call failed with %d' % retcode, output)
+            exitcode_file = os.path.join(INPUT_DIR, self.name + '.exitcode')
```

Comment:
This will catch cases where it returns a non-zero error code when it shouldn't, but not the other way around. We should check for the exitcode (retcode) file regardless.

Reply:
Done.